### PR TITLE
feat: websocket functionality in browser agent

### PR DESCRIPTION
**Pull request by harshvardhan-212**
- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
[https://held-warlock-e75.notion.site/WebSocket-Implementation-for-Browser-Agent-in-WebApp-15e07c41ab5880b4aa6bfbf62d2e2955]

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: packages/@sparrow-workspaces/src/features/socket-explorer/layout/SocketExplorer.svelte [hunk 1]
```diff
@@ -195,6 +195,7 @@
                         <div class="h-100 d-flex flex-column">
                           <ResponsePreviewNavigator
                             {webSocket}
+                            isWebApp={true}
```
Comment (Astitva877) [inline packages/@sparrow-workspaces/src/features/socket-explorer/layout/SocketExplorer.svelte:198]: don't pass this isWebApp = true directly from here, take this from pages and pass here.

Comment (harshvardhan-212) [reply packages/@sparrow-workspaces/src/features/socket-explorer/layout/SocketExplorer.svelte:198]: ok

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: packages/@sparrow-workspaces/src/features/socket-explorer/components/response-preview-navigator/ResponsePreviewNavigator.svelte [hunk 2]
```diff
@@ -194,14 +195,17 @@
...
-          onClick={handleDownloaded}
-          disable={false}
-          loader={false}
-        />
-      </Tooltip>
+
+      {#if isWebApp === true}
+        <Tooltip title={"Export"}>
+          <WithButtonV4
+            icon={DownloadIcon2}
```
Comment (itsmdasifraza) [inline packages/@sparrow-workspaces/src/features/socket-explorer/components/response-preview-navigator/ResponsePreviewNavigator.svelte:202]: when webapp is set to true don't show export item.

File: packages/@sparrow-workspaces/src/features/graphql-explorer/components/response-navigator/ResponseNavigator.svelte [hunk 3]
```diff
@@ -132,6 +132,7 @@
           />
         </Tooltip>
         <!-- Download button -->
+        <!-- Download button -->
         <Tooltip title={"Export"}>
           <WithButtonV6
             icon={DownloadIcon2}
```
Comment (itsmdasifraza) [inline packages/@sparrow-workspaces/src/features/graphql-explorer/components/response-navigator/ResponseNavigator.svelte:138]: duplicate content

**Review by itsmdasifraza — CHANGES_REQUESTED** [review]

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**